components/project-card/meta/creators.jsx

```diff
@@ -7,18 +7,39 @@ const Creators = (props) => {
 
   let classnames = classNames(`creator d-block open-sans`, props.className);
   let labelText = props.showLabelText ? `Created by ` : ``;
+  let joinedCreators;
 
-  return <p className="my-2"><small className={classnames}>{labelText}{props.creators.join(`, `)}</small></p>;
+  if (typeof props.creators[0] === `string` ) {
+    joinedCreators = props.creators.join(`, `);
+  } else {
+    let creators = [];
+    for (let creator of props.creators){
+      // So that creators which are without a profile aren't links
+      let url = null;
+      if(typeof creator.profile_id === `number` && props.makeLink) {
+        url = `/profile/${creator.profile_id}`;
+      }
+      creators.push(<a key={creators.length} href={url}>{creator.name}</a>);
+    }
+    // Because react doesn't like to render comma separated JSX objects easily
```

Comment:
Because it was rendering as `[Object object], [Object object]` or something like that. This could be my JSX is rusty and I can't remember the rules around rendering things- maybe I shouldn't be using an array of JSX nodes?